### PR TITLE
Question sidebar visual polish

### PR DESCRIPTION
**Pull request by mazameli**
- Makes the action buttons look a bit nicer
- Nicer spacing between buttons + description vs the activity feed
- Less bold Revert button
- Airier spacing between question title and breadcrumbs
- Last-edited info is now same size and style as data breadcrumbs

Requesting a review from @kdoh because I bet he could dial in the spacing on some of these elements more nicely than I've been able to — I ran into a couple places, like the space betwee the question title and the chevron icon, where I wasn't sure how to tweak spacing with the styled-system approach we're now using.

image

### CODE REVIEW

File: frontend/src/metabase/components/LastEditInfoLabel.js [hunk 1]
```diff
@@ -45,11 +45,11 @@ function LastEditInfoLabel({ item, user, onClick, ...props }) {
     editorId === user.id ? t`you` : formatEditorName(first_name, last_name);
 
   return (
-    <TextButton
+    <Badge
```
Comment (daltojohnso) [inline frontend/src/metabase/components/LastEditInfoLabel.js:48]: The `Badge` component ends up rendering an `<a>` tag which really should only be used for links. Things that are actions like this really ought to be `<button>` tags, even if they are styled to look like links. I added the `TextButton` styled component to try to handle this use case, so can you try continue using `TextButton` and either tweak the css of it or do this:

```
const TextButtonWithAddedStyles = TextButton.extend`
  css-property: foo;
  padding: 1rem;
`

```

And then you can us that as a react component.

**Review by daltojohnso — CHANGES_REQUESTED** [review]
I can take care of any broken tests, but can you try to keep using `TextButton`? I'm a stickler when it comes to using anchor and button tags 🙂

File: frontend/src/metabase/components/LastEditInfoLabel.js (hunk 1 above)
Comment (mazameli) [reply frontend/src/metabase/components/LastEditInfoLabel.js:48]: Cool, thanks for the correction! What I tried to do initially was apply our `text-small` class to that instance of `TextButton` via `className`, but it wasn't seeming to work. I'll see if I can make that work or use that `extend` syntax.

File: frontend/src/metabase/query_builder/components/QuestionActivityTimeline.jsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ export function QuestionActivityTimeline({
 
   return (
     <div className={className}>
-      <SidebarSectionHeader>{t`Activity`}</SidebarSectionHeader>
+      <SidebarSectionHeader>{t`Revision history`}</SidebarSectionHeader>
```
[the remark this answers is not in the document]
Comment (mazameli) [reply frontend/src/metabase/query_builder/components/QuestionActivityTimeline.jsx]: Fair question. I was trying to move away from the urgency of "activity" but I've revised this to just "History" which will still be accurate when BUCM items get woven in.

**Comment by mazameli** [comment]
@daltojohnso, LMK if the way I got TextButton to do what I wanted it to is kosher with you.

**Review by daltojohnso — APPROVED** [review]
LGTM

**Comment by mazameli** [comment]
Gonna merge this and let @kdoh comment on your `dj-question-sidebar` branch if he has further suggestions.

**Pull request merged**